lib/model.js

```diff
@@ -1852,18 +1853,18 @@ class Model {
   }
 
   /**
-   * Search for a single instance by its primary key._
```

Comment:
Many changes here due to whitespacing. Maybe it's better to do those in a different PR if you really want them? Now it just adds more things to review in an already big PR